lib/ansible/plugins/loader.py

```diff
@@ -271,19 +271,11 @@ def _get_paths(self, subdirs=True):
         # PLUGIN_PATHS_CACHE, and MODULE_CACHE.  Since those three dicts key
         # on the class_name and neither regular modules nor powershell modules
         # would have class_names, they would not work as written.
-        reordered_paths = []
-        win_dirs = []
-
-        for path in ret:
-            if path.endswith('windows'):
-                win_dirs.append(path)
-            else:
-                reordered_paths.append(path)
-        reordered_paths.extend(win_dirs)
+        ret.sort(key=lambda p: p.endswith('/windows'))
```

Comment:
Should that be `ret.sort(key=lambda p: p.endswith('windows'))` to match the logic before?

Reply:
I think either would work. I just put the `/` in there. Some example data:
```
 ('/Users/sdoran/Source/ansible/lib/ansible/modules/windows', 'setup.ps1'),
 ('/Users/sdoran/Source/ansible/lib/ansible/modules/windows', 'win_hosts.ps1'),
```